### PR TITLE
Event: provide verbose comment for focus(in | out) & rename support prop

### PR DESCRIPTION
**Pull request by markelog**

### CODE REVIEW

File: src/event.js [hunk 1]
```diff
@@ -753,15 +753,21 @@ jQuery.each({
...
-// Create "bubbling" focus and blur events
-if ( !support.focusinBubbles ) {
+// Support: Firefox
+// Firefox doesn't have focus(in | out) events
+// Related ticket - https://bugzilla.mozilla.org/show_bug.cgi?id=687787
+//
+// Support: Chrome, Safari
+// focus(in | out) events fire after focus & blur events,
+// which is spec violation - http://www.w3.org/TR/DOM-Level-3-Events/#events-focusevent-event-order
+// Related ticket - https://code.google.com/p/chromium/issues/detail?id=449857&q=focusin&colspec=ID%20Pri%20M%20Week%20ReleaseBlock%20Cr%20Status%20Owner%20Summary%20OS%20Modified
```
Comment (mzgol) [inline src/event.js]: Everything after "id=number" should be cut off

Comment (markelog) [inline src/event.js]: Sure, although i would prefer you would use optative then imperative mood

Comment (mzgol) [inline src/event.js]: "...can be cut off"? ;)

Comment (markelog) [inline src/event.js]: Thank you :-), i appreciated it.

**Comment by mzgol** [comment]
It's nice to have more info :+1:

**Comment by markelog** [comment]
It just took me couple hours to realize that this shim wasn't there to create bubble focus(in | out) events :-)

**Comment by dmethvin** [comment]
Anything that helps people understand this mess is a good thing. :+1:

**Pull request closed without merging**